### DISCUSSION TITLE
How to download a non static file, which computed during run time?

### DISCUSSION THREAD

**Discussion by soumendrak** [fastapi/fastapi#7417] [Questions] [answered]
### First check

* [x] I added a very descriptive title to this issue.
* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [x] I already read and followed all the tutorial in the docs and didn't find an answer.
* [x] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
* [x] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [x] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [x] After submitting this, I commit to one of:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * I already hit the "watch" button in this repository to receive notifications and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

**Comment by phy25** [comment, marked as answer]
https://fastapi.tiangolo.com/advanced/custom-response/#streamingresponse

**Comment by soumendrak** [comment]
Thanks, able to do with https://fastapi.tiangolo.com/advanced/custom-response/#fileresponse

**Comment by tiangolo** [comment]
Thanks for the help here everyone @phy25 ! :clap: :bow: 

Thanks for reporting back and closing the issue :+1:
